client/src/pages/OpeningDashboardPage/CreateOpeningPage.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import Sidebar from "../../components/Sidebar";
+import { createOpenings } from "./../../api/opening";
+import { useSelector } from "react-redux";
+
+const CreateOpeningPage = () => {
+  const userId = useSelector((state) => state.auth.user._id) || null;
+
+  const [formData, setFormData] = useState({
+    company: "",
+    number: 1,
+  });
+  const [validMessage, setValidMessage] = useState("");
+
+  const handleChange = (e) => {
+    setFormData({
+      ...formData,
+      [e.target.id]: e.target.value,
+    });
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    if (formData.number < 1) {
+      setValidMessage("Number must be greater than 0");
+      return;
+    }
+
+    createOpenings(formData, userId);
+    toast.success("Form successfully submitted");
+  };
+
+  return (
+    <div className="flex">
+      <Sidebar />
+
+      <main className="flex-1 flex flex-col items-center p-4">
+        <h1 className="text-3xl">Create Openings</h1>
+        <br></br>
+        <form onSubmit={handleSubmit}>
+          <div class="mb-6">
+            <label
+              for="company"
+              class="block mb-2 text-sm font-medium text-gray-900"
+            >
+              Company:
+            </label>
+            <input
+              type="company"
+              id="company"
+              placeholder="Google"
+              value={formData.company}
+              onChange={handleChange}
+              class="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
```

Comment:
again, className can't be this long. I also have className for input text field, you can have a look in index.css to use it